src/modules/number/index.ts

```diff
@@ -443,4 +443,88 @@ export class NumberModule extends SimpleModuleBase {
 
     return min + offset;
   }
+
+  /**
+   * Returns a roman numeral in [String](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#string_type) format.
+   * The bounds are inclusive.
+   *
+   * @param options Maximum value or options object.
+   * @param options.min Lower bound for generated roman numerals. Defaults to `1`.
+   * @param options.max Upper bound for generated roman numerals. Defaults to `3999`.
+   *
+   * @throws When `min` is greater than `max`.
+   * @throws When `min`, `max` is not a number.
+   * @throws When `min` is less than `1`.
+   * @throws When `max` is greater than `3999`.
+   *
+   * @example
+   * faker.number.romanNumeral() // "CMXCIII"
+   * faker.number.romanNumeral(5) // "III"
+   * faker.number.romanNumeral({ min: 10 }) // "XCIX"
+   * faker.number.romanNumeral({ max: 20 }) // "XVII"
+   * faker.number.romanNumeral({ min: 5, max: 10 }) // "VII"
+   *
+   */
+
+  romanNumeral(options: number | { min?: number; max?: number } = {}): string {
+    const DEFAULT_MIN = 1;
+    const DEFAULT_MAX = 3999;
+
+    let num;
+    let result = '';
+
+    if (typeof options === 'number') {
+      options = {
+        max: options,
+      };
+    }
+
+    const min = Number(options.min ?? DEFAULT_MIN);
+    const max = Number(options.max ?? DEFAULT_MAX);
+
+    if (typeof min !== 'number' || isNaN(min)) {
+      throw new FakerError('Min value must be a valid number.');
+    }
+
+    if (typeof max !== 'number' || isNaN(max)) {
+      throw new FakerError('Max value must be a valid number.');
+    }
+
+    if (min < DEFAULT_MIN) {
+      throw new FakerError(`Min value ${min} should be 1 or greater.`);
+    }
+
+    if (max > DEFAULT_MAX) {
+      throw new FakerError(`Max value ${max} should be 3999 or lesser.`);
+    }
+
+    if (max < min) {
+      throw new FakerError(`Max ${max} should be larger then min ${min}.`);
+    }
+
+    num = this.int({ min, max });
```

Comment:
Please declare variables late/shortly before using it.

```suggestion
    let num = this.int({ min, max });
    let result = '';
```